### PR TITLE
Fix the formatting and clean up roadctl examples

### PR DESCRIPTION
**Pull request by MarkGreenPR**
README and cheatsheet markdown files, templates.txt and workerPool.txt

### CODE REVIEW

File: microservices/ga/datamgr/README.md [hunk 1]
```diff
@@ -13,110 +16,133 @@ The build system requires git source code management.  If the directory you choo
...
 Git push doesn't include tags.  To push tags to the origin use:
+
     git push origin --tag
 
 ## roadctl
-The roadctl is modeled after kubectl. Use roadctl help for a list of top
-level commands.  This may be different for a specific command or it's 
-sub-commands.
+The roadctl command is modeled after kubectl.
+Use "roadctl help" for a list of top level commands.
```
Comment (jscharber) [inline microservices/ga/datamgr/README.md:53]: Actually just typing roadctl without help does the same thing

File: microservices/ga/datamgr/README.md [hunk 2]
```diff
@@ -13,110 +16,133 @@ The build system requires git source code management.  If the directory you choo
...
-          --debug string    Debug level: info(default)|warm|error|critical (default "d")
-      --format string   Output format: text(default)|json|yaml (default "f")
-  -h, --help            help for roadctl
+Usage:
+  roadctl [command]
+
+Available Commands:
+  apply       Apply configuration to named resource
+  completion  Generate completion scripts on stdout
+  config      manage roadctl global configuration options
```
Comment (jscharber) [inline microservices/ga/datamgr/README.md]: I thought we were going to change all the Usage commands to start with uppercase, aka Manage roadctl global config...

File: microservices/ga/datamgr/README.md [hunk 3]
```diff
@@ -13,110 +16,133 @@ The build system requires git source code management.  If the directory you choo
...
+      --templates string   Set the location of the directory holding roadctl templates
+      --token string       OAUTH access token
+      --user string        HTTP basic auth user name
 
 Use "roadctl [command] --help" for more information about a command.
+```
 
 ### Specific command
-roadctl get --help
+    roadctl get --help
```
Comment (jscharber) [inline microservices/ga/datamgr/README.md:106]: Syntax is backwords
```
roadctl help get
```

File: microservices/ga/datamgr/cheatsheet.md [hunk 4]
```diff
@@ -60,18 +60,18 @@
 
 # Resources
 
-Can be accessed using their short name or full name.  Multiple resources can be requested depending on the kubernetes service/control being used.
+Can be accessed using their short name or full name.  Multiple resources can be requested depending on the Kubernetes service/control being used.
 
 ```bash
-kubectl get po,deploy,svc 
+kubectl get pod,deploy,svc
```
Comment (jscharber) [inline microservices/ga/datamgr/cheatsheet.md:66]: po is the short name for pod

File: microservices/ga/datamgr/cheatsheet.md [hunk 5]
```diff
@@ -158,7 +158,7 @@ status: {}
 $ kubectl create -f dnsutil.yaml 
 pod/dnsutils created
 
-$ kubectl get po -o wide
+$ kubectl get pod -o wide
```
Comment (jscharber) [inline microservices/ga/datamgr/cheatsheet.md:161]: same as above, I was using short names for all the examples so I think it is consistent

File: microservices/ga/workerPool/README.md [hunk 6]
```diff
@@ -42,11 +42,11 @@ are installed:
 - A .gitignore with entries to exclude the .templates directory
 - An initialized templates repository
 
-It doesn't generate a git tag; pick your own. Once preflight.sh 
+It doesn't generate a git tag; pick your own. Once preflight.sh
 successfully executes, a ".pr_preflight_check" is created.  Until this
 file exists, Make will always attempt to run preflight.sh.
 
-If you are running a version prior to v6.0.0alpha, you may run preflight.sh 
+If you are running a version prior to v6.0.0alpha, you may run preflight.sh
```
Comment (jscharber) [inline microservices/ga/workerPool/README.md]: Version is wrong, it should be v0.6.0alpha

File: microservices/ga/workerPool/README.md [hunk 7]
```diff
@@ -160,34 +160,37 @@ This template generates the following endpoints:
...
 aggregated response for any Metric interfaces defined plus the standard
-metrics for the dispatcher.  The jobs and scheduler endpoints support 
-modifying the jobs currently defined or changing the schedule of the 
+metrics for the dispatcher.  The jobs and scheduler endpoints support
+modifying the jobs currently defined or changing the schedule of the
 scheduler.
 
 # Git
-The build system requires git source code management.  If the directory you choose to generate your service is not under git control, do the following after executing your template.
+The build system requires git source code management.
```
Comment (jscharber) [inline microservices/ga/workerPool/README.md]: Preflight handles this, I think we can just mention it and kill the rest off.  The .templates directory doesn't exist anymore.

File: microservices/ga/workerPool/README.md [hunk 8]
```diff
@@ -197,93 +200,112 @@ Git push doesn't include tags.  To push tags to the origin use:
...
+      --password string    HTTP basic auth password
+      --templates string   Set the location of the directory holding roadctl templates
+      --token string       OAUTH access token
+      --user string        HTTP basic auth user name
 
 Use "roadctl [command] --help" for more information about a command.
+```
 
 ### Specific command
-roadctl get --help
```
Comment (jscharber) [inline microservices/ga/workerPool/README.md:243]:
```
roadctl help get
```

**Review by jscharber — CHANGES_REQUESTED** [review]

File: microservices/ga/datamgr/cheatsheet.md (hunk 4 above)
Comment (MarkGreenPR) [reply microservices/ga/datamgr/cheatsheet.md:66]: I was just going for readability, all of them can easily be changed back

File: microservices/ga/workerPool/README.md (hunk 7 above)
Comment (MarkGreenPR) [reply microservices/ga/workerPool/README.md]: just mention that preflight.sh does this then?

**Comment by MarkGreenPR** [comment]
#42

**Pull request merged**